lib/presentation/crowd_action/pages/crowd_action_details.dart

```diff
@@ -1100,7 +1050,7 @@ class _BadgesPopupCard extends StatelessWidget {
                           maxWidth: 300,
                         ),
                         child: Text(
-                          'Short description about how you can earn different achievements/badges',
+                          'Different commitments give you a different set of points. These points correspond with badges.\n Which one will you attain this month?',
```

Comment:
Remove space after `\n`

Reply:
![image](https://user-images.githubusercontent.com/55907631/145702589-43aa1e1c-b38e-4656-8610-af7ce76f407f.png)
![image](https://user-images.githubusercontent.com/55907631/145702593-6614d61c-500b-48f6-85d1-e30ed6f5e25c.png)

Reply:
Again, I'm not talking about removing `\n`, I'm talking about removing this ` ` that comes after the `\n`